content-repo/extra-docs/articles/Phishing-Campaign.md

```diff
@@ -4,91 +4,124 @@ title: Phishing Campaign
 description: How to detect and manage phishing campaigns in Cortex XSOAR using the Phishing Campaign Content Pack.
 ---
 
+# Phishing Campaign #
 
-## Phishing Campaign
-
-A phishing campaign is a collection of phishing incidents that originate from the same attacker, or as part of the same organized attack launched against multiple users.
+The Phishing Campaign pack enables you to find, create and manage phishing campaigns. A phishing campaign is a collection of phishing incidents that originate from the same attacker, or as part of the same organized attack launched against multiple users.
 
 As phishing campaigns are a number of phishing incidents that are similar to each other, it is important to detect and create the links between them, and look at them as a whole, rather than spend time investigating each incident separately. To see how to set up a phishing incident generally in Cortex XSOAR, go to the [Phishing Use Case Tutorial](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-2/cortex-xsoar-tutorials/tutorials/phishing-use-case.html).
 
-To detect and manage phishing campaigns in Cortex XSOAR, go to **Marketplace** and install the **Phishing Campaign** Content Pack. 
 
 ### How It Works
 
-The following flow chart describes the architecture of phishing campaigns in Cortex XSOAR:
+ The following flow chart describes the architecture of phishing campaigns in Cortex XSOAR:
 
 ![image](https://user-images.githubusercontent.com/43602124/124762458-97eeb480-df3b-11eb-9479-2214037befea.png)
 
-The **Phishing Campaign** Content Pack contains the following content:
+Included in this content pack is the **Detect & Manage Phishing Campaigns** playbook. Use this playbook in the [Phishing Investigation Generic V2 playbook](https://xsoar.pan.dev/docs/reference/playbooks/phishing-investigation---generic-v2), or use it in your custom phishing playbook. As part of the phishing incident, the playbook does the following: 
+ - Finds and links related incidents to the same phishing attack (a phishing campaign).
+ - Searches for an existing Phishing Campaign incident or creates a new incident for the linked Phishing incidents.
+ - Links all detected phishing incidents to the Phishing Campaign incident that was found or that was previously created.
+ - Updates the Phishing Campaign incident with the latest data about the campaign, and also updates all related phishing incidents to indicate that they are part of the campaign.
+
+
+
+## In This Pack ##
+
+The **Phishing Campaign** content pack contains several content items.
+
+### Automations ###
```

Comment:
@richardbluestone there's another automation: `IsIncidentPartOfCampaign`.
That automation takes the list of incidents detected as similar by the `FindEmailCampaign` script, and then checks whether one of them is already linked to a `Phishing Campaign` incident. If it is - it will output the ID of that incident so that all the similar phishing incidents can be linked to it too.
We use that script to find whether there's an existing campaign incident or we need to create a new one.